components/header/Header.tsx

```diff
@@ -1,35 +1,175 @@
-import React, { useState, FC } from "react";
-
-import { Link } from "../Link";
-import Logo from "../../public/logo.svg";
-import NavLinks from "./NavLinks";
-import MenuToggle from "./MenuToggle";
-import MobileMenu from "./MobileMenu";
-import {
-  Wrapper,
-  MaxWidth,
-  Navigation,
-  MobileNavigation,
-} from "./Header.styled";
-
-export const Header: FC = () => {
+/* eslint-disable react/display-name */
+import React from "react";
+import { useRouter } from "next/router";
+import { DownIcon } from "../Icons";
+import * as UI from "./Header.styled";
+import Image from "next/image";
+import { useState } from "react";
+import { COMMUNITY_LINKS } from "utils";
+
+const NavLinks = () => {
+  const router = useRouter();
+
+  return (
+    <UI.LinkList>
+      {HEADER_LINKS.map((link) => (
+        <UI.LinkListItem key={link.key}>
+          {link.component({ path: router.pathname })}
+        </UI.LinkListItem>
+      ))}
+    </UI.LinkList>
+  );
+};
+
+export const Header: React.FunctionComponent = () => {
   const [showMobileMenu, setShowMobileMenu] = useState(false);
+
+  const onToggle = () => {
+    setShowMobileMenu((prevShowMobileMenu) => !prevShowMobileMenu);
+  };
+
   return (
-    <Wrapper>
-      <MaxWidth>
-        <Link href="https://umaproject.org">
-          <Logo />
-        </Link>
-        <Navigation>
+    <UI.Container>
+      <UI.Content>
+        <UI.LogoLink href="/">
+          <UI.LogoLinkIcon src="/logo.svg" alt="UMA logo" />
```

Comment:
should this go to umaproject or stay on umaverse?

Reply:
hmm, good question